app/scripts/controllers/transactions/index.js

```diff
@@ -2654,4 +2654,16 @@ export default class TransactionController extends EventEmitter {
   _getApprovalId(txMeta) {
     return String(txMeta.id);
   }
+
+  _addUnapprovedTxsApprovals() {
+    const unapprovedTxs = this.txStateManager.getUnapprovedTxList();
+    Object.values(unapprovedTxs).forEach((txMeta) => {
+      this._requestApproval(txMeta, false);
+    });
+  }
+
+  addInitialTransactionApprovals(callback) {
```

Comment:
Perhaps just `initApprovals` as `Transaction` is implied, and it keeps it generic for any future usage?